### ISSUE TITLE
Allow only image load!

### ISSUE THREAD

**Issue by jsamuel** [RequestPolicy/requestpolicy#138]
```
imported trac ticket
created: 2010-09-30 00:52:57
reporter: Cattleya
```

I think it would be better if it has a feature that allow image load normally, can you add filter to allow image load in the whitelist, thanks!
Because I'm using ImgLIkeOpera to block image, if I want to allow image, I must do two steps, unblock by ImgLikeOpera and add this image site to whitelist.

**Comment by jsamuel** [comment]
```
imported trac comment
created: 2011-02-20 11:55:42
author: eibwen
```

Duplicate of #61, particularly given ticket:61#comment:3.
